model/src/main/kotlin/Vulnerability.kt

```diff
@@ -45,30 +45,52 @@ data class Vulnerability(
     val url: URL? = null
 ) {
     /**
-     * The rating attaches human-readable semantics to the score number, see
-     * https://www.first.org/cvss/specification-document#Qualitative-Severity-Rating-Scale.
+     * The rating attaches human-readable semantics to the score number according to CVSS version 2, see
+     * https://www.first.org/cvss/v2/guide.
```

Comment:
Can you add a source for the ranges?

Reply:
I actually didn't find one on the official CVSS pages for v2 (only for v3 I found the ranges to be explicitly defined). The best source for the ranges (or comparison of ranges) I've found is the link I've mentioned in the commit message. I preferred to not use that link also here, as it's less official, but I could do so if you prefer.

Reply:
Yeah that was exactly the reason why I asked.
IMO it's better to add it here as well, as that source is better than nothing, and future code readers will be less confused where these values come from.

Reply:
Ok, done.